packages/terra-form-select/CHANGELOG.md

```diff
@@ -3,6 +3,27 @@ ChangeLog
 
 Unreleased
 ----------
+### Breaking Changes
+* Removed boundingRef
+* Removed isPlaceholderHidden - The placeholder option is no longer added by default.
+* Removed name - Terra recommends handling validations with [react-final-form](https://github.com/final-form/react-final-form).
+* Removed required - Terra recommends handling validations with [react-final-form](https://github.com/final-form/react-final-form).
```

Comment:
I'm on the fence removing this, I think it may still be valuable just for setting aria-required on the element with the `role="listbox"` or equivalent element in the variations.

Reply:
I'm curious if that's something we can do. Technically, input elements are the only ones that should have that attached, and I'm not sure if we have any elements attached here that could be used.

Reply:
Yeah, that's a good point, I'm fine with it as is.